assets/js/aplus.js

```diff
@@ -86,6 +86,37 @@ $(function() {
 
     $(window).bind('scroll', modifyMenu);
     $(window).bind('resize', updateMenu);
+
+    /**
+    * Warn about links that open in new windows.
+    */
+
+    function addLinkType(link, type) {
+      let linkTypes = (link.getAttribute('rel') || '').split(' ');
+      if (!linkTypes.includes(type)) {
+        linkTypes.push(type);
+      }
+      link.setAttribute('rel', linkTypes.join(' ').trim());
+    }
+
+    function addExternalLinkIcon(link) {
+      if (!link.querySelector('.icon')) {
+        link.insertAdjacentHTML('beforeend', `<span class="icon external-link"></span>`);
+      }
+    }
+
+    function addScreenReaderMessage(link, message) {
+      if (!link.querySelector('.screen-reader-only')) {
+        link.insertAdjacentHTML('beforeend', `<span class="screen-reader-only"> (${message})</span>`);
+      }
+    }
+
+    document.querySelectorAll('a[target="_blank"]').forEach(link => {
+      console.log("open in a new tab");
```

Comment:
`console.log` should be removed.

Reply:
When I was searching for my `console.log()` I found some others in the code. How many of those are intentional?
![image](https://user-images.githubusercontent.com/10437475/99817847-21d94100-2b56-11eb-8753-d0a2d9d293f0.png)

Reply:
Let's ignore the old console.logs now, but your new code does not need it.